var/spack/repos/builtin/packages/octopus/package.py

```diff
@@ -36,10 +37,10 @@ class Octopus(Package):
     depends_on('libxc@2:2.99', when='@:5.99')
     depends_on('libxc@2:3.99', when='@6:7.99')
     depends_on('libxc@2:4.99', when='@8:9.99')
-    depends_on('libxc@3:5.0.0', when='@10.0')
+    depends_on('libxc@3:5.0.0', when='@10.0:')
     depends_on('mpi')
     depends_on('fftw@3:+mpi+openmp', when='@:9.99')
-    depends_on('fftw-api@3:', when='@10.0:')
+    depends_on('fftw-api@3:', when='@10.5:')
```

Comment:
Can you briefly comment on this change? Was `fftw` not needed before?

Reply:
Octopus uses the fftw-api package.  Maybe I don't understand what you're asking me....

Reply:
I think I see what you're saying, and, yes I think it was an error!  I've fixed this to require fftw-api when the version is >= 10.